contracts/eosio.msig/eosio.msig.cpp

```diff
@@ -20,6 +20,9 @@ void multisig::propose() {
    constexpr size_t max_stack_buffer_size = 512;
    size_t size = action_data_size();
    char* buffer = (char*)( max_stack_buffer_size < size ? malloc(size) : alloca(size) );
+   struct { 
+      struct S { ~S()  { if (buf) free(buf); } char *buf; } s; 
+   } __freemem{max_stack_buffer_size<size?buffer:0};
```

Comment:
Not needed, this will be freed at end of action.
Please submit changes to this contract to the https://github.com/EOSIO/eosio.contracts repo.